packages/renderer/src/lib/diagnose/DiagnosePage.svelte

```diff
@@ -0,0 +1,13 @@
+<script>
+import NavPage from '../ui/NavPage.svelte';
+import DiagnoseDevToolsConsoleLogs from './DiagnoseDevToolsConsoleLogs.svelte';
+import DiagnosePageProviders from './DiagnosePageProviders.svelte';
+</script>
+
+<NavPage title="Self-diagnosis" searchEnabled="{false}">
```

Comment:
```suggestion
<NavPage title="Self diagnostic" searchEnabled="{false}">
```

Reply:
I've renamed the whole content from 'Diagnose' to 'Troubleshooting'